### ISSUE TITLE
Suggestions corrections names populated places, rivers and lakes relating to the letter ë

### ISSUE THREAD

**Issue by Maxszik** [nvkelso/natural-earth-vector#617]
Corrections names populated places, rivers and lakes relating to the letter ë. To to use the letter ë in names and name_en is fishy and I believe not correct:

ne_10m_rivers_lake_centerlines_scale_rank: ne_id 1159120867 Olëkma. Should be Olyokma. (Local name in Russian is Олёкма). - [Wiki](https://en.wikipedia.org/wiki/Olyokma) [Wikidata](https://www.wikidata.org/wiki/Q26358)

In ne_10m_rivers_lake_centerlines and ne_50m_rivers_lake_centerlines:
Olenëk ne_id 1159117003. Name should be Olenyok. - [Wiki](https://en.wikipedia.org/wiki/Olenyok_(river)) [Wikidata](https://www.wikidata.org/wiki/Q26799)

In ne_10m_lakes:
Ozero Chërnoye ne_id 1159114929 with name_alt: Lake Chërnoye ought to be spelled simply Ozero (or Lake) Chornoye.

In ne_50m_lakes:
Ozero Chërnoye ne_id  1159114929 (same as above) has name_alt: L. Chërnoye.
In other words, it differs from the name_alt of the 10 m dataset. That doesn't make sense?

In ne_10m_populated_places:
Zvëzdnyj in ne_10m_populated_places_simple called Zvezdnyy. The correct name would be Zvyozdny. - [Wiki](https://en.wikipedia.org/wiki/Zvyozdny_(inhabited_locality)) [Wikidata](https://www.wikidata.org/wiki/Q24559914)
